Calculator/CalculatorBrain.swift

```diff
@@ -12,7 +12,7 @@ enum OperatorType: String {
     case addition = "+"
     case subtraction = "-"
     case multiplication = "×"
-    case division = "/"
+    case division = "÷"
 }
 
```

Comment:
created other cases in the enum that's needed for the calcualtor. had to use the storyboard's calculator's operator texts to make the calculator fully functional, so I copied and pasted from that into the rawValues' strings.